### PR TITLE
Add GrapheneOS

### LINKED ISSUE #9280: GrapheneOS
### Brand Name

GrapheneOS

### Website

https://grapheneos.org/

### Popularity Metric

Similarweb rank: 41,212

### Official Resources for Icon and Color

svg: https://grapheneos.org/780febcc.mask-icon.svg
color: #00000 - eyedropped from the svg icon

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by SebiTalent04**
GrapheneOS

**Issue:** closes #9280

**Similarweb rank:** 41,212
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

Tried not adding all the commits from my other icon but no luck

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8101,6 +8106,11 @@
             "hex": "BA478F",
             "source": "https://metabrainz.org/projects"
         },
+        {
+            "title": "Musixmatch",
+            "hex": "FF3E59",
+            "source": "https://www.musixmatch.com"
+        },
```
Comment (adamrusted) [inline _data/simple-icons.json:8113]:
```suggestion
```

File: icons/musixmatch.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Musixmatch</title><path d="M21.6.09c1.19 0 2.4.84 2.4 2.15v19.58c0 1.3-1.21 2.19-2.4 2.19-.51 0-1.04-.16-1.49-.53 0 0-4.03-3.23-8.11-6.52l-8.19 6.52c-.45.37-.98.53-1.49.53-1.19 0-2.32-.88-2.32-2.19V2.33C0 1.03 1.13.14 2.33.14c.51 0 1.04.16 1.49.53l8.19 6.47s.05-.04.07-.06C14.75 4.93 20.11.62 20.11.62c.45-.36.98-.53 1.49-.53ZM12 7.14l5.97 4.91-5.97 4.9-6.01-4.9L12 7.14Zm-6.01 4.91L0 16.89V7.22l5.99 4.84ZM24 7.14v9.81l-6.03-4.9L24 7.14Zm0-4.91v4.91l-6.03 4.91L12 7.14 20 .7c1.71-1.53 4-.33 4 1.53ZM12 7.14l-6.01 4.91L0 7.22V2.29C0 .43 2.32-.58 3.82.63L12 7.13Z"/></svg>
```
Comment (adamrusted) [inline icons/musixmatch.svg:1]:
```suggestion
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
You'll need to remove MusixMatch from this PR - will take a look once we've got one icon in here.

**Comment by SebiTalent04** [comment]
Why did the check failed? Does it have something to do with the "OS" in the name?

**Comment by adamrusted** [comment]
Looks like it's because the `source` is listed twice. Remove one of them and it should be fine!

File: _data/simple-icons.json [hunk 3]
```diff
@@ -5180,6 +5180,11 @@
             "source": "https://graphql.org/brand",
             "guidelines": "https://graphql.org/brand"
         },
+        {
+            "title": "GrapheneOS",
+            "hex":"000000",
+            "source":"https://grapheneos.org"
```
Comment (SebiTalent04) [inline _data/simple-icons.json:5186]: @adamrusted it is listed once but the linter sees two of each `hex` and `source`

Comment (adamrusted) [reply _data/simple-icons.json:5186]: Weird. Looks like it's not in the correct place alphabetically (currently after GraphQL) - that might be causing some of this weirdness.

**Comment by SebiTalent04** [comment]
Finally!

File: icons/grapheneos.svg
Comment (adamrusted) [inline icons/grapheneos.svg:1]: Hey @SebiTalent04 - it looks like you've [stretched the icon](https://github.com/simple-icons/simple-icons/assets/1830867/affe2250-1495-4c6f-aad9-658ffd0f7fde) to fit the 24*24 viewbox. Could you rebuild from source, and make sure it looks more like the red version in that screenshot?

**Comment by PeterShaggyNoble** [comment]
Any update on this, @SebiTalent04?

File: icons/grapheneos.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>GrapheneOS</title><path d="m6.25 7.24 3.94-1.97c-.07-.19-.12-.39-.12-.61 0-.84.69-1.52 1.6-1.67V0h.66v3c.91.15 1.6.84 1.6 1.67 0 .22-.05.42-.13.61l3.95 1.97c.14-.15.32-.27.52-.38.83-.42 1.86-.24 2.45.38l2.96-1.5.33.51-2.96 1.5c.32.76-.02 1.63-.85 2.05-.19.09-.37.15-.57.19v4.01c.2.04.38.1.57.19.83.42 1.17 1.29.85 2.05l2.96 1.5-.33.51-2.96-1.5c-.59.62-1.62.8-2.45.38-.2-.11-.37-.24-.52-.38-1.31.65-2.63 1.32-3.95 1.97.08.19.13.39.13.61 0 .84-.69 1.52-1.6 1.67v3h-.66v-3c-.91-.15-1.6-.84-1.6-1.67 0-.22.05-.42.12-.61l-3.94-1.97c-.14.15-.32.27-.52.38-.83.42-1.86.24-2.45-.38l-2.96 1.5-.33-.51 2.96-1.5c-.32-.76.02-1.63.85-2.05.19-.09.37-.15.57-.19V10c-.2-.04-.38-.1-.57-.19-.83-.42-1.17-1.29-.85-2.05l-2.96-1.5.33-.51 2.96 1.5c.59-.62 1.62-.8 2.45-.38.2.11.37.24.52.38Zm11.16 9c-.19-.45-.16-.98.14-1.43.31-.47.84-.76 1.41-.84V9.99c-.57-.07-1.1-.36-1.41-.84-.3-.44-.33-.97-.14-1.43l-3.95-1.98c-.36.36-.88.59-1.47.59s-1.11-.23-1.47-.59L6.57 7.72c.19.45.16.98-.14 1.43-.31.47-.84.76-1.41.84v3.98c.57.07 1.1.36 1.41.84.3.44.33.97.14 1.43l3.95 1.98c.36-.36.88-.59 1.47-.59s1.11.23 1.47.59l3.95-1.98Z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/grapheneos.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>GrapheneOS</title><path d="m6.953 7.244 3.458-1.97a1.901 1.901 0 0 1-.108-.608c0-.835.608-1.525 1.407-1.67V0h.58v2.995a1.704 1.704 0 0 1 1.407 1.67c0 .219-.045.418-.118.609l3.468 1.97c.127-.146.281-.273.454-.382a1.705 1.705 0 0 1 2.151.382l2.596-1.498.29.508-2.595 1.498a1.69 1.69 0 0 1-1.244 2.242v4.012a1.69 1.69 0 0 1 1.244 2.242l2.596 1.498-.29.508-2.597-1.498a1.705 1.705 0 0 1-2.151.382 2.065 2.065 0 0 1-.454-.382c-1.153.654-2.306 1.317-3.468 1.97.073.19.118.39.118.608 0 .835-.608 1.525-1.407 1.67V24h-.58v-2.995a1.704 1.704 0 0 1-1.407-1.67c0-.219.045-.418.108-.609l-3.458-1.97a2.065 2.065 0 0 1-.454.382 1.705 1.705 0 0 1-2.151-.382l-2.596 1.498-.29-.508 2.595-1.498a1.69 1.69 0 0 1 1.244-2.242V9.994a1.69 1.69 0 0 1-1.244-2.242L1.461 6.254l.29-.508 2.597 1.498a1.705 1.705 0 0 1 2.151-.382c.173.11.327.236.454.382Zm9.803 9.004a1.682 1.682 0 0 1 .128-1.425 1.702 1.702 0 0 1 1.234-.835v-3.976a1.702 1.702 0 0 1-1.234-.835 1.682 1.682 0 0 1-.128-1.425L13.29 5.773c-.318.363-.772.59-1.289.59-.517 0-.971-.227-1.289-.59L7.244 7.752c.163.454.136.98-.128 1.425a1.702 1.702 0 0 1-1.234.835v3.976c.5.073.962.363 1.234.835.264.445.291.971.128 1.425l3.467 1.979c.318-.363.772-.59 1.289-.59.517 0 .971.227 1.289.59z"/></svg>
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]

**Review by service-paradis — APPROVED** [review]

**Pull request merged**